### PR TITLE
SDV2: update sdv2 benchmarking

### PR DESCRIPTION
**Pull request by Songyuanwei**
# What does this PR do?

Update the benchmark for sd inference.

Use inference/sd_infer.py to test performance.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/mindspore-lab/mindone/blob/master/CONTRIBUTING.md)?
- [ ] Did you make sure to update the documentation with your changes? E.g. record bug fixes or new features in `What's New`. Here are the
      [documentation guidelines](https://github.com/mindspore-lab/mindcv/wiki/%E6%96%87%E6%A1%A3%E7%BC%96%E5%86%99%E8%A7%84%E8%8C%83)
- [ ] Did you build and run the code without any errors?
- [ ] Did you report the running environment (NPU type/MS version) and performance in the doc? (better record it for data loading, model inference, or training tasks)
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@xxx

### CODE REVIEW

File: examples/stable_diffusion_v2/benchmark.md [hunk 1]
```diff
@@ -59,17 +59,18 @@ Flash Attention,
...
 |---------------|:-----------|:------------:|:------------------:|:----------------:|:----------------:|:----------------:|:----------------:|
-| 1.5           |     D910x1-MS2.0    |  DPM++       |   20       |    512x512         |       4          |    2.50        |           0.40   |
-| 2.0           |     D910x1-MS2.0    |  DPM++       |   20       |    512x512         |       4          |    2.86       |        0.44       |
-| 2.1-v         |     D910x1-MS2.0    |  DPM++       |   20       |    768x768         |       4          |     1.18      |         0.19      |
-| 1.5           |     D910*x1-MS2.2   |  DPM++       |   20       |    512x512         |       4          |       2.50     |         0.39      |
-| 2.0           |     D910*x1-MS2.2   |  DPM++       |   20       |    512x512         |       4          |      2.86     |         0.42      |
-| 2.1-v         |     D910*x1-MS2.2   |  DPM++       |   20       |    768x768         |       4          |      1.67     |         0.25      |
+| 1.5           |     D910x1-MS2.2.10    |  DDIM       |   20       |    512x512         |       4          |    3.27        |       0.58       |
```
Comment (zhtmike) [inline examples/stable_diffusion_v2/benchmark.md:62]: DDIM usually needs 30~50 steps, 20 steps' result is not good enough

File: examples/stable_diffusion_v2/benchmark.md [hunk 2]
```diff
@@ -59,17 +59,18 @@ Flash Attention,
...
+| 2.1-v         |     D910x1-MS2.2.10    |  DDIM       |   20       |    768x768         |       4          |     1.09     |         0.20      |
+| 1.5           |     D910*x1-MS2.2.10   |  DDIM       |   20       |    512x512         |       4          |       5.56     |         0.87      |
+| 2.0           |     D910*x1-MS2.2.10   |  DDIM       |   20       |    512x512         |       4          |      7.80     |         1.14      |
+| 2.1-v         |     D910*x1-MS2.2.10   |  DDIM       |   20       |    768x768         |       4          |      2.51     |         0.40      |
 > Context: {Ascend chip}-{number of NPUs}-{mindspore version}.
 > Speed (step/s): sampling speed measured in the number of sampling steps per second.
 > FPS (img/s): image generation throughput measured in the number of image generated per second.
 
 Note that the performance of SD2.1 should be similar to SD2.0 since they have the same network architecture.
+Multi card parallel performance is the same as single card performance.
```
Comment (HaoyangLee) [inline examples/stable_diffusion_v2/benchmark.md:73]: 是指多卡时每张卡的性能等于单卡时的性能？

Comment (Songyuanwei) [reply examples/stable_diffusion_v2/benchmark.md:73]: 是的

**Comment by zhtmike** [comment]
And also can try `MS_ENABLE_GE=1` on MS910* currently, there is a visible speed improvement

File: examples/stable_diffusion_v2/benchmark.md (hunk 2 above)
Comment (HaoyangLee) [reply examples/stable_diffusion_v2/benchmark.md:73]: 英文表述有点歧义

File: examples/stable_diffusion_v2/benchmark.md (hunk 1 above)
Comment (Songyuanwei) [reply examples/stable_diffusion_v2/benchmark.md]: done, now DDIM use 30 steps.

File: examples/stable_diffusion_v2/benchmark.md (hunk 2 above)
Comment (Songyuanwei) [reply examples/stable_diffusion_v2/benchmark.md]: done

**Comment by Songyuanwei** [comment]
> And also can try `MS_PYNATIVE_GE` on MS910* currently, there is a visible speed improvement

No improvement for sdv2

**Review by zhtmike — APPROVED** [review]

**Review by SamitHuang — APPROVED** [review]

**Pull request merged**